docs/getting_started/trying_wagtail.rst

```diff
@@ -1,78 +0,0 @@
-==============
```

Comment:
Well the one-liners are now dead, so by "all" you mean just the command-line instructions - yes they're in the repo docs.